### ISSUE TITLE
select: label accessibility

### ISSUE THREAD

**Issue by nikkimk** [patternfly/patternfly-elements#2664]
## Description of the issue*

- `pf-listbox` should have an accessible label
- `pf-select`'s toggle button text should not be the item selected but something more descriptive

* Note this breaks with upstream API but is more accessible.

### Impacted component(s)

- `pf-listbox`
- `pf-select`

### Steps to reproduce

1. Go to [Select documentation](https://patternflyelements.org/components/select/)
2. Inspect the first example.
3. Check the accesisbility tree.

### Expected behavior

Listbox and toggle button should have descriptive labels.

**Comment by bennypowers** [comment]
> pf-select's toggle button text should not be the item selected but something more descriptive

we can add an aria-label to the button, but the select element shouldn't look or behave differently to upstream
